docs/scripts_md/tarchive_validation.md

```diff
@@ -0,0 +1,74 @@
+# NAME
+
+tarchive\_validation.pl -- Validates the tarchive against the one inserted in
+the LORIS database.
+
+# SYNOPSIS
+
+perl tarchive\_validation.pl `[options]`
+
+Available options are:
+
+\-profile     : name of the config file in ../dicom-archive/.loris-mri
+
+\-reckless    : upload data to the database even if the study protocol
+                is not defined or if it is violated
+
+\-globLocation: loosen the validity check of the tarchive allowing for
+                the possibility that the tarchive was moved to a
+                different directory
+
```

Comment:
one minor comment: indentation; see screenshot when I run perldoc at the terminal:
![tarchive_validation](https://user-images.githubusercontent.com/15198379/36264748-983eb942-123b-11e8-9d1c-c751c66689ec.png)